src/lib/components/chat/ChatWindow.svelte

```diff
@@ -38,6 +40,7 @@
 
 	let loginModalOpen = false;
 	let message: string;
+	let timeout: ReturnType<typeof setTimeout>;
 
```

Comment:
```suggestion
	let isSharedRecently = false;
	$: $page.params.id && (isSharedRecently = false);
```

Reply:
handled in https://github.com/huggingface/chat-ui/pull/645/commits/fdbb4cafbf846c95557cb7e2fe0ce45effcd4c76